Services/Container/classes/Screen/MemberViewLayoutProvider.php

```diff
@@ -36,7 +36,7 @@ public function isInterestedInContexts() : ContextCollection
      */
     public function getPageBuilderDecorator(CalledContexts $screen_context_stack) : ?PageBuilderModification
     {
-        if (!$screen_context_stack->current()->hasReferenceId()) {
+        if (!$screen_context_stack->current()->hasReferenceId() || $this->dic["lti"]->isActive()) {
```

Comment:
I'm not a fan of being dependent on LTI-Code here. Isn't there another solution? For example, if we extend the GlobalScreen so that you can not only log in for contexts, but also explicitly log out?

Reply:
I see that issue but i am not sure if i understand what you mean with a "GS logout" extension . The intention of transfering the responsibility of memberview "rendering" to the LTI PageBuilderModifiction was to avoid a priority collision with LTI Implementation. The base intention of using a PageBuilderModifiction instead of single MetaBar/MainBar/Footer in LTI was to avoid the collision with MainBarModification with the new Help Service.

By taking a loook to the new ModeInfo in MemberView i wonder why this is rendered by the whole page component (page->withModeInfo()) even though Mainbar, Metabar and footer are not affected. Maybe it would be sufficient to surround the content frame content->withModeInfo() for the memberview?, or why not an implementation of flexible ModeInfo: content->withModeInfo page->withModeInfo.. just my thoughts.